docs/index.md

```diff
@@ -79,3 +79,58 @@ information.
   </div>
 </div>
 </div>
+
+## What is Melange
+
+Melange is a comprehensive toolset comprising several key components:
```

Comment:
```suggestion
Melange is a set of tools that come together to generate and interoperate with modern JavaScript:
```

Reply:
good explanation overall, but the first sentence threw me off a little bit in the beginning.